validation/chOpenApiValidator.go

```diff
@@ -54,14 +55,15 @@ func NewCHValidator(openApiSpec string) (CHValidator, error) {
 
 	// Successfully created a CHValidator so return the fully constructed object.
 	return CHValidatorImpl{
-		doc: doc,
+		doc:         doc,
+		openApiSpec: openApiSpec,
 	}, nil
 }
 
-// ValidateRequestAgainstOpenApiSpec takes a request and an openAPI spec location (string relative path) and uses the
-// spec to validate the provided request. If any validation errors are found, then they are formatted and returned to the
-// caller. If any errors are encountered while attempting to validate, they are handled and also returned to the caller.
-func (chv CHValidatorImpl) ValidateRequestAgainstOpenApiSpec(httpReq *http.Request, openApiSpec, contextId string) ([]byte, error) {
+// ValidateRequestAgainstOpenApiSpec takes a request and uses an openAPI3 spec to validate the it. If any validation errors
```

Comment:
Maybe expand on the "to validate the it"! :D Previously it was "to validate the provided request" but maybe put request body

Reply:
Changed to mention the request's body as it's more descriptive